sagemaker_model_monitor/SageMaker-ModelMonitoring-Batch-Transform-Model-Quality-With-SageMaker-Pipelines-On-schedule.ipynb

```diff
@@ -0,0 +1,882 @@
+{
```

Comment:
<p>Please delete duplicate notebooks and data under sagemaker_model_monitor directory. Only keep the copies in the <code>sagemaker_model_monitor/model_monitor_batch_transform</code> directory</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3639/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
done